Soulgram.Interests/Soulgram.Interests.Application/Commands/Movies/AddMovieCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MediatR;
+using Soulgram.Interests.Application.Models.Request.Movies;
+
+namespace Soulgram.Interests.Application.Commands.Movies;
+
+public class AddMovieCommand : IRequest
+{
+    public AddMovieCommand(AddMovieRequest request)
+    {
+        Request = request;
+    }
+
+    public AddMovieRequest Request { get; }
```

Comment:
In my opinion, it does not make sense to divide a commands class and its handlers

Reply:
> I suppose you do not need this property because IRequest provide it by default

Sorry I didn't catch 



> In my opinion, it does not make sense to divide a commands class and its handlers

Hmmm, mb let's discuss